app/src/main/AndroidManifest.xml

```diff
@@ -101,7 +101,10 @@
         </activity>
 
         <activity android:name=".ui.onboarding.OnboardingActivity" />
-        <activity android:name=".ui.onboarding.BridgeWizardActivity" />
+        <activity
+            android:name=".ui.onboarding.BridgeWizardActivity"
+            android:configChanges="orientation|screenSize" />
```

Comment:
Setting `android:configChanges` here prevents the `BridgeWizardActivity` from getting destroyed and recreated upon device rotation. This means that the cancellation of any outstanding network requests in its `onDestroy()` will be bypassed when the device rotates since the user is still in the `BridgeWizardActivity`.